### ISSUE TITLE
Code signing "Beacon.framework" failed 💩

### ISSUE THREAD

**Issue by andreyz** [helpscout/beacon-ios-sdk#113]
* Beacon install method: [ ] Carthage, [ ] CocoaPods, [x] Manual

Beacon SDK 2.0.2, Xcode 12.0 (12A7209)

**Actual outcome**
Beacon prevented app archive being accepted for submission to AppStore Connect. This is blocking my iOS 14 app update...

**Expected outcome**
Beacon shouldn't prevent app from being submitted to the AppStore. The least it could do, actually.

**Screenshots or GIFs**
<img width="324" alt="image" src="https://user-images.githubusercontent.com/205317/93764698-dbc63680-fc13-11ea-8643-44e6697e552a.png">
```
2020-09-21 12:05:15 +0000  Skipping architecture thinning for item "Beacon" because arch "arm64e" wasn't found
2020-09-21 12:05:15 +0000  Skipping stripping extended attributes of item: ...
2020-09-21 12:05:15 +0000  Entitlements for <IDEDistributionItem: xxxx; bundleID='net.helpscout.beacon', 
2020-09-21 12:05:15 +0000  Writing entitlements for <IDEDistributionItem: xxx; bundleID='net.helpscout.beacon', 2020-09-21 12:05:15 +0000  /var/folders/17/6_kd877j6t7bmn12kzfv9kp80000gn/T/XcodeDistPipeline.~~~VbgQjk/Root/Payload/XXX.app/Frameworks/Beacon.framework: replacing existing signature
2020-09-21 12:05:16 +0000  /var/folders/17/6_kd877j6t7bmn12kzfv9kp80000gn/T/XcodeDistPipeline.~~~VbgQjk/Root/Payload/XXX.app/Frameworks/Beacon.framework: code object is not signed at all
```

**Comment by andreyz** [comment]
I see, this is a shortcoming of the SDK documentation for manual integration instructions. The i386 architecture needs stripping from Beacon.framework if to be successfully submitted to the store.

This [strip-frameworks.sh script](https://github.com/krzyzanowskim/ObjectivePGP/blob/055b1ec8ecb8eaddb9c3b58cbfc038467b28ec87/scripts/strip-frameworks.sh) added as a build phase resolved the issue.

**Comment by ScottPetit** [comment]
Hey @andreyz,

Yea this is an unfortunate side effect of having frameworks building for device and simulator.  We assumed the link to Carthage instructions in our [manual frameworks section](https://github.com/helpscout/beacon-ios-sdk#manually) would help but I'll take a look at updating them for clarity.

**Comment by andreyz** [comment]
@ScottPetit "integrate it similar to how Carthage integrates frameworks" isn't at all clear, especially if it links to steps on how to use Cartage to integrate frameworks.

**Comment by ScottPetit** [comment]
Hey 👋,

We're going to update our docs and make the manual steps much clearer.  We're tracking this internally [here](https://helpscout.atlassian.net/browse/IOSSDK-362) and will update when it's been updated.

**Comment by andreyz** [comment]
This issue is no longer relevant after 2.1.0 release

**Comment by ScottPetit** [comment]
Hi @andreyz We discovered an issue with version 2.1.0 where under certain configurations Beacon wouldn't be able to be dismissed. Therefore we decided to remove 2.1.0 and aim to resolve this issue in the next release. We'll update you once this is released, thank you for your patience.

**Comment by ScottPetit** [comment]
Hey @andreyz, 

This should be resolved once again with the latest version (2.1.1).  Feel free to take a look at the [Changelog](https://github.com/helpscout/beacon-ios-sdk/blob/master/CHANGELOG.md) for more info.
